assets/src/edit-story/elements/text/edit.js

```diff
@@ -353,6 +354,11 @@ function TextEdit({
     cssVar: '--faux-selection-color',
     expectedStyle: 'color',
   });
+  useColorTransformHandler({
+    id,
+    targetRef: wrapperRef,
+    expectedStyle: 'background',
+  });
```

Comment:
This now breaks for highlight - the background suddenly changes color instantly, and the actual highlight color only changes debounced. And this addition also required for border color for the text edit component, by the way.

Reply:
Fixed in 851e4c5